### PR TITLE
re #1633: finish preprint->repository refactor for searching preprints

### PR DESCRIPTION
**Pull request by hardyoyo**
- use correct object models throughout
- tweak search form to use a name for the search_term input
- tweak list page to indicate the search_term if it's present

### CODE REVIEW

File: src/repository/views.py [hunk 1]
```diff
@@ -207,50 +208,48 @@ def repository_list(request, subject_slug=None):
...
-        )
+        # from_author = models.Preprint.objects.filter(
+        #     (Q(preprintauthor__in=split_search_term))
+        # )
 
-        articles_from_author = [article for article in submission_models.Article.preprints.filter(
-            authors__in=from_author,
-            stage=submission_models.STAGE_PREPRINT_PUBLISHED,
-            date_published__lte=timezone.now())]
+        # preprints_from_author = [preprint for preprint in models.Preprint.objects.filter(
```
Comment (hardyoyo) [inline src/repository/views.py:235]: Oops... I forgot that I'd commented the author and institution search to get this working... that remains to be done.

Comment (hardyoyo) [reply src/repository/views.py:235]: I think I'm in over my head with this... I can't seem to make progress on searching by author name or institution... we may want to defer that functionality for a future ticket?

**Comment by ajrbyers** [comment]
I'll take a look the AM.

**Comment by hardyoyo** [comment]
superseded by #1766, closing this

**Pull request closed without merging**